lib/streamlit/hashing.py

```diff
@@ -60,7 +61,46 @@
 NP_SAMPLE_SIZE = 100000
 
 
-Context = namedtuple("Context", ["globals", "cells", "varnames"])
+# "None"sense as a placeholder for literal None object while hashing.
+NONESENSE = b"streamlit-4w350m3-None"
+
+# Arbitrary item to denote where we found a cycle in a hashed object.
+# This allows us to hash self-referencing lists, dictionaries, etc.
+CYCLE_PLACEHOLDER = b"streamlit-57R34ML17-hesamagicalponyflyingthroughthesky"
```

Comment:
How about

"streamlit-57R34ML17-hesamagicalponyflyingthroughthesky-None"
and
"streamlit-57R34ML17-hesamagicalponyflyingthroughthesky-Cycle"